src/LogzioShipper/json_parser.py

```diff
@@ -0,0 +1,25 @@
+import logging
+import json
+
+from typing import Generator
+from .file_parser import FileParser
+
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+class JsonParser(FileParser):
+
+    def __init__(self, file_data: str) -> None:
+        super().__init__(file_data)
+
+    def parse_file(self) -> Generator:
+        jsons = self.file_data.split('\n')
```

Comment:
are assuming that all json are minified ?

Reply:
Yeah. It looks like from code json will be one line so it should be ok (if I am not wrong Roni did the same)